lora/fuse.py

```diff
@@ -66,6 +66,10 @@
 
     model.update_modules(tree_unflatten(fused_linears))
     weights = dict(tree_flatten(model.parameters()))
+
+    if quantization := config.get("quantization", None):
+        config.pop("quantization")
+
```

Comment:
Maybe I miss something, but I don't think that's correct... we requantize the model after it's fused?

https://github.com/ml-explore/mlx-examples/blob/main/lora/models.py#L120-L125

Reply:
@awni  You are right. Sorry, I didn't notice that.

Reply:
@awni Any thoughts on making the requantize linear optional, so people can fuse models in fp16 and requantize them using other advanced quantization methods such as GPTQ, AWQ, GGUF?

Reply:
Yes that would be totally fine... you probably are aware but that would also require dequantizing the rest of the model (not just the LoRA layers).

Reply:
Yeah, it's a bit tricky for non full layers lora..